dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -1,7 +1,8 @@
 require 'pd/certificate_renderer'
 
-class Pd::WorkshopMailer < ActionMailer::Base
+class Pd::WorkshopMailer < ApplicationMailer
   include Rails.application.routes.url_helpers
+  include ActionMailerMetrics
```

Comment:
You don't need this since it's already included in the parent class `ApplicationMailer`